low-level/framework/rabbitmq/rabbitmq_egress_processor.py

```diff
@@ -296,6 +296,10 @@ def _transmit_msg_on_exchange(self):
                                             routing_key=self._routing_key,
                                             properties=msg_props,
                                             body=jsonMsg)
+
+                    # RAAL stands for RAise ALert
+                    logger.info(f"RAAL: {jsonMsg}")
```

Comment:
Just change line 311 to logger.info, retaining existing content, as it applies/works for actuator also
Avoid "RAAL" & related comment